chapters.md

```diff
@@ -3,31 +3,84 @@
 
 # Chapters
 
-## Edition and Chapter Flags
+The Matroska Chapters system can have multiple `Editions` and each `Edition` can consist of `Simple Chapters` where a chapter start time is used as marker in the timeline only. An `Edition` can be more complex with `Ordered Chapters` where a chapter end time stamp is additionally used or much more complex with `Linked Chapters`. The Matroska Chapters system can also have a menu structure, borrowed from the DVD menu system, or have it's own Native Matroska menu structure.
```

Comment:
The DVD menu system is one example of multiple possibilities. We should not mention in a generic paragraph. It's a chapter codec, and just like the other codecs, should/will be in another document.